### ISSUE TITLE
Missing ARIA attributes

### ISSUE THREAD

**Issue by marvel-uiuc** [web-illinois/ilw-section-nav#1]
There are some ARIA attributes missing, at least `aria-controls` and `aria-expanded`. Go over best practices to check for other missing ones as well.
